tests/themes/run.sh

```diff
@@ -12,30 +12,20 @@ if ! [ -f CMakeCache.txt ]; then
   exit 64
 fi
 build_dir="$PWD"
-source_dir="$(dirname -- "$0")/../.."
+tests_dir="$(dirname -- "$0")/.."
 
 config_file="$build_dir/test-themes-awesomerc.lua"
 
-exit_code=0
 for theme in themes/*/theme.lua; do
   echo "== Testing $theme =="
   theme=${theme%/*}
   theme=${theme##*/}
 
   sed "s~default/theme~$theme/theme~g" "awesomerc.lua" > "$config_file"
 
-  output=$(AWESOME_RC_FILE="$config_file" \
+  AWESOME_RC_FILE="$config_file" \
     AWESOME_THEMES_PATH="$PWD/themes" \
     AWESOME_ICON_PATH="$PWD/icons" \
     VERBOSE=1 \
-    "$source_dir/tests/run.sh" themes/tests.lua \
-    | grep -Ev '^(awesome_log:|== Running|1 tests finished.|.{19} W: awesome: (a_glib_poll|Cannot reliably detect EOF)|xterm: cannot load font|Running step)' || true)
-  if [ -n "$output" ]; then
-    echo "== Unexpected output: =="
-    echo "$output"
-    echo "========================"
-    exit_code=1
-  fi
+    "$tests_dir/run.sh" themes/tests.lua
```

Comment:
Should this be squashed with the original commit which added `tests/themes/run.sh`?